Tests/NodeVisitor/NodeActivatorTest.php

```diff
@@ -25,18 +25,16 @@ class NodeActivatorTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->request = $this
```

Comment:
`$this->request` and `$this->activator` not needed anymore? Do we need the setup method at all then? :blush: